libr/lang/lang.c

```diff
@@ -174,6 +174,8 @@ R_API RLangPlugin *r_lang_get_by_name (RLang *lang, const char *name) {
 		if (!strcasecmp (h->name, name)) {
 			return h;
 		}
+		if (h->trivial_name && !strcasecmp(h->trivial_name, name))
```

Comment:
several issues here
* strcasecmp is not portable
* missing space before (
* missing braces around the if block